gravitee-apim-e2e/ui-test/support/e2e.ts

```diff
@@ -21,4 +21,4 @@ import './common/api.commands';
 
 // Alternatively you can use CommonJS syntax:
 // require('./commands')
-require('cypress-terminal-report/src/installLogsCollector')({ enableExtendedCollector: Cypress.env('printLogsToConsole') === 'always' });
+// require('cypress-terminal-report/src/installLogsCollector')({ enableExtendedCollector: Cypress.env('printLogsToConsole') === 'always' });
```

Comment:
🗑️  ?

Reply:
Alright, I'll remove that line and will also de-install the cypress-terminal-report module as it created more problems that benefits.